threedi_modelchecker/config.py

```diff
@@ -1176,221 +1142,215 @@ def is_none_or_empty(col):
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            ~is_none_or_empty(models.GlobalSetting.initial_waterlevel_file),
-            models.GlobalSetting.water_level_ini_type == None,
+        column=models.InitialConditions.initial_water_level_aggregation,
+        invalid=Query(models.InitialConditions).filter(
+            ~is_none_or_empty(models.InitialConditions.initial_water_level_file),
+            models.InitialConditions.initial_water_level_aggregation == None,
         ),
-        message="an initial waterlevel type (v2_global_settings.water_level_ini_type) should be defined when using an initial waterlevel file.",
+        message="an initial waterlevel type (initial_settings.initial_water_level_aggregation) should be defined when using an initial waterlevel file.",
     ),
     QueryCheck(
         error_code=323,
-        column=models.GlobalSetting.maximum_table_step_size,
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            models.GlobalSetting.maximum_table_step_size
-            < models.GlobalSetting.table_step_size,
+        column=models.ModelSettings.maximum_table_step_size,
+        invalid=Query(models.ModelSettings).filter(
+            models.ModelSettings.maximum_table_step_size
+            < models.ModelSettings.minimum_table_step_size,
         ),
-        message="v2_global_settings.maximum_table_step_size should be greater than v2_global_settings.table_step_size.",
+        message="model_settings.maximum_table_step_size should be greater than model_settings.minimum_table_step_size.",
     ),
     QueryCheck(
         error_code=325,
         level=CheckLevel.WARNING,
-        column=models.GlobalSetting.interception_global,
-        invalid=Query(models.GlobalSetting).filter(
-            first_setting_filter,
-            ~is_none_or_empty(models.GlobalSetting.interception_file),
-            is_none_or_empty(models.GlobalSetting.interception_global),
+        column=models.Interception.interception,
+        invalid=Query(models.Interception).filter(
+            ~is_none_or_empty(models.Interception.interception_file),
+            is_none_or_empty(models.Interception.interception),
         ),
-        message="v2_global_settings.interception_global is recommended as fallback value when using an interception_file.",
+        message="interception.interception is recommended as fallback value when using an interception_file.",
     ),
 ]
 
 CHECKS += [
-    QueryCheck(
-        error_code=326,
-        level=CheckLevel.INFO,
-        column=table.id,
-        invalid=Query(table).filter(
-            table.id != Query(setting).filter(first_setting_filter).scalar_subquery()
-        ),
-        message=f"{table.__tablename__} is defined, but not referred to in v2_global_settings.{setting.name}",
+    UsedSettingsPresentCheck(
+        error_code=326, level=CheckLevel.ERROR, column=use_col, settings_table=table
```

Comment:
This is a different kind of error so you shouldn't reuse the error code.

Reply:
Actually changed the number of `MaxOneRecordCheck` because that one is new; see 7d322027cbe2a1340eb1bbe1b910316e95f1eb6f